worlds/pokemon_crystal/options.py

```diff
@@ -23,11 +23,13 @@ class Goal(Choice):
     """
     Elite Four: Defeat the Champion and enter the Hall of Fame
     Red: Defeat Red at Mt. Silver
+    Diploma: Reach your Dexcount goal and Print your Diploma
```

Comment:
```suggestion
    Diploma: Reach your Dexcount goal and obtain the diploma in Celadon City
```
The event is just to signify you _can_ print the diploma, not that you've done so